spec/controllers/qing_groups_controller_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'spec_helper'
+
+describe QingGroupsController, type: :request do
+  before do
+    @user = create(:user, role_name: 'coordinator')
+    @form = create(:form, question_types: %w(text text))
```

Comment:
If you do like `question_types: ['text', ['text', 'text']]`, you will get a qing group for free. You can get it via `@form.children[1]` or `@form.c[1]`.